server/consumer.go

```diff
@@ -2543,7 +2543,12 @@ func (o *consumer) infoWithSnapAndReply(snap bool, reply string) *ConsumerInfo {
 
 	// If we are replicated and we are not the leader we need to pull certain data from our store.
 	if rg != nil && rg.node != nil && !o.isLeader() && o.store != nil {
-		state, _ := o.store.BorrowState()
+		state, err := o.store.BorrowState()
+		if err != nil {
+			o.mu.Unlock()
+			o.srv.Warnf("Error retrieving consumer state from store: %s", err)
```

Comment:
This can happen naturally as a simple race from a consumer being deleted vs consumer info call. Should not be a warn IMO, actually would not even log anything here. This can be normal.

Reply:
Fair enough, will remove the log line.